packages/website/components/superhot.js

```diff
@@ -0,0 +1,256 @@
+import Img from '../components/cloudflareImage'
+import kwesforms from 'kwesforms'
+import Circles from './circles'
+import CheckMark from './checkMark'
+import { useEffect } from 'react'
+
+export default function Superhot() {
+  useEffect(() => {
+    kwesforms.init()
+  }, [])
+
+  return (
+    <>
+      <div className="circle-container overflow-hidden">
+        <Circles />
+      </div>
+      <div className="bg-peach relative overflow-hidden text-center">
+        <div className="circle-container overflow-hidden -mt-[50px] lg:mt-[-70px]">
+          <Circles />
+        </div>
+        <div className="mx-auto max-w-7xl w-full pt-16 pb-20 lg:py-32 mt-12">
+          <div className="relative px-6">
+            <h1 className="font-open-sans text-xl lg:text-2xl leading-snug font-bold text-black ">
+              <span className="block">
+                Want more speed? Check out our premium features!
+              </span>
+            </h1>
+          </div>
+
+          <div className="px-6 max-w-5xl mx-auto mt-12">
+            <div className="relative  before:bg-[#F5C32C] before:absolute before:w-full before:h-full before:left-4 before:-top-4">
+              <div className="relative bg-white p-6 sm:p-10 lg:p-16 border-2 border-black">
+                <Img
+                  src="images/superhot-logo.svg"
+                  alt="logo"
+                  className="mb-12 mx-auto block"
+                />
+
+                <h4 className="text-lg md:text-xl leading-snug lg:text-2xl font-semibold max-w-4xl mx-auto">
+                  Permacache NFT data of up to 5GB on the edge to guarantee the
```

Comment:
```suggestion
                  Permacache NFT files of up to 5GB on the edge to guarantee the
```

Data feels like the entire account limit, and not file limit